mockiato-codegen/src/generate.rs

```diff
@@ -19,17 +17,41 @@ mod lifetime_rewriter;
 mod mock_struct;
 mod trait_impl;
 
-pub(crate) fn generate_mock(
-    mockable_attr: MockableAttr,
-    item_trait: &ItemTrait,
-    trait_decl: &TraitDecl,
-) -> TokenStream {
-    let mock_struct_ident = mock_struct_ident(&trait_decl, mockable_attr.name_attr);
+#[derive(Debug, Default)]
+pub(crate) struct GenerateMockOptions {
+    pub(crate) mock_struct_ident: Option<Ident>,
```

Comment:
Naming suggestion: `custom_struct_ident`

Reply:
Addressed in 67d2d7c